docs/cli.md

```diff
@@ -176,38 +195,54 @@ Examples:
+| &minus;&minus;handler                          | Invoke the function handler inside the code file                                                        |
+| &minus;&minus;mode                             | Special run mode ('pass' for using the command as is)                                                   |
+| &minus;&minus;schedule                         | Cron schedule                                                                                           |
+| &minus;&minus;from&minus;env                   | Read the spec from the env var                                                                          |
+| &minus;&minus;dump                             | Dump run results as YAML                                                                                |
+| &minus;&minus;image                            | Container image                                                                                         |
+| &minus;&minus;kind                             | Serverless runtime kind                                                                                 |
+| &minus;&minus;source                           | Source code archive/git                                                                                 |
+| &minus;&minus;local                            | Run the task locally (ignore runtime)                                                                   |
+| &minus;&minus;auto&minus;mount                 | Add volume mount to job using auto mount option                                                         |
+| &minus;&minus;workdir                          | Run working directory                                                                                   |
+| &minus;&minus;origin&minus;file                | For internal use                                                                                        |
+| &minus;&minus;label                            | Run labels (key=val)                                                                                    |
+| -w, &minus;&minus;watch                        | Watch/tail run log                                                                                      |
+| &minus;&minus;verbose                          | Verbose log                                                                                             |
+| &minus;&minus;scrape&minus;metrics             | Whether to add the `mlrun/scrape-metrics` label to this run's resources                                 |
+| &minus;&minus;env&minus;file                   | Path to .env file to load config/variables from                                                         |
+| &minus;&minus;auto&minus;build                 | When set functions will be built prior to run if needed                                                 |
```

Comment:
| &minus;&minus;auto&minus;build | When set, the function image will be built prior to run if needed